utils/unittest/execution_state.go

```diff
@@ -24,7 +24,7 @@ const ServiceAccountPrivateKeySignAlgo = crypto.ECDSAP256
 const ServiceAccountPrivateKeyHashAlgo = hash.SHA2_256
 
 // Pre-calculated state commitment with root account with the above private key
-const GenesisStateCommitmentHex = "abdc5bc5456726872d3b3a99f70eedafe7f198736e7f5d44c77a7c12f4b84ba5"
+const GenesisStateCommitmentHex = "9410871291ff9aa6d1a0267fc87fb20db2fb2bc575014aac8df135a42992389a"
```

Comment:
This change is due to 1 more read per transaction.

Reply:
1 more read or one more right? cause I don't think reads are actually changing anything about the state commitments